### ISSUE TITLE
Display of multiple seals in single charter preview duplicates information

### ISSUE THREAD

**Issue by GVogeler** [icaruseu/mom-ca#1001]
The XML `<cei:sealDesc><cei:seal>1) des <cei:sigillant>Ebf. Poppo</cei:sigillant> aufgedrückt</cei:seal> und <cei:seal>2) des <cei:sigillant>Propst Adalbero</cei:sigillant>, aufgedrückt</cei:seal></cei:sealDesc>` contains two seals with individual sigillants. Preview of a saved charter (endpoint `/saved-charter`) displays this as follows:
![grafik](https://user-images.githubusercontent.com/324006/103405097-385fb880-4b56-11eb-899d-ebb33c8ed5a5.png)
Error is probably part of https://github.com/icaruseu/mom-ca/blob/59703228e9de2b222be9439fc61cdd530a8c78bd/my/XRX/src/mom/app/charter/xsl/cei2html.xsl#L962-L1014.

**Comment by GVogeler** [comment]
I would suggest refactoring this part to use xsl:template and xsl:apply-templates methods instead of xsl:choose clauses:
```
<xsl:template match="cei:sealDesc"><xsl:choose><xsl:when test="text()/normalize-space()!=''><xsl:value-of select="."/></xsl:when><xsl:otherwise><xsl:apply-templates/></xsl:otherwise></xsl:template>
<xsl:template match="cei:seal"><p>Seal: <xsl:apply-templates/></xsl:template>
<xsl:template match="cei:sigillant"><span>Sigillant:</span> <xsl:apply-templates/></xsl:template>
...
```
should come closer to the expected outcome (if sealDesc contains text without markup, then just use is as plain text; if it describes the information in separate elements (cei:seal, cei:sigillant etc.) then use a display with captions)

**Comment by StephanMa** [comment]
Dear @GVogeler 
which charter is this?
For testing purposes...

**Comment by NTsch** [comment]
Closed by #1062.
